src/components/webview/webviewEvents.ts

```diff
@@ -162,4 +162,9 @@ export type WebviewResponse =
 	| Readonly<{
 			kind: 'webview.global.flipCodemodHashDigest';
 			codemodNodeHashDigest: CodemodNodeHashDigest;
+	  }>
+	| Readonly<{
+			kind: 'webview.jobDiffView.contentModified';
```

Comment:
I think we should call `jobDiffView` just `panel` because it shares job diffing and codemod description.